### PR TITLE
update repo in pom files and update readme with new table names

### PR DESCRIPTION
**Pull request by albertshau**
for Wise app.

### CODE REVIEW

**Comment by anwar6953** [comment]
Alright, just apply the pom changes to MovieRecommender, and then we can close: https://github.com/caskdata/cdap-apps/pull/76

**Comment by albertshau** [comment]
o actually they should all have the non-snapshot too, let me change

**Comment by anwar6953** [comment]
ok.
Also, http://builds.cask.co/build/admin/edit/editBuildTasks.action?buildKey=CA-WAUT-JOB1
The Goal is simply:
clean package

This should probably include a -U, though I was unable to make the mvn command fail locally (even with -U), unless I removed /m2/repository

File: TwitterSentiment/pom.xml [hunk 1]
```diff
@@ -47,8 +47,9 @@
 
   <repositories>
     <repository>
-      <id>apache.snapshots</id>
-      <url>https://repository.apache.org/content/repositories/snapshots</url>
+      <id>oss.sonatype</id>
+      <name>OSS Sonatype Public</name>
+      <url>http://oss.sonatype.org/content/groups/public/</url>
```
Comment (chtyim) [inline TwitterSentiment/pom.xml:52]: Why need this? Isn't that everything should be in maven central?

Comment (anwar6953) [inline TwitterSentiment/pom.xml:52]: Both Ajai and I were getting problems building some apps in cdap-apps (finding the cdap 2.8 snapshot dependencies), until we put this in.

**Pull request merged**